stm32_base/hal/imu_stm32/invensense_9150.c

```diff
@@ -16,6 +21,14 @@ static int is9150_readreg(uint8_t reg_addr, uint8_t *reg_val)
     return res;
 }
 
+static int is9150_write_reg(uint8_t reg_addr, uint8_t reg_val)
+{
+    int res = 0;
```

Comment:
why 3 lines?  Why not just `return i2c_write_reg8(is9150_dev.i2c, is9150_dev.addr, reg_addr, reg_val);`